README.md

```diff
@@ -1,30 +1,30 @@
-![](http://magentys.github.io/donut/img/Donut-05.png) 
+![](http://donutreport.github.io/donut/img/Donut-05.png) 
 
-[![Build Status](https://travis-ci.org/MagenTys/donut.svg?branch=master)](https://travis-ci.org/MagenTys/donut)
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.magentys/donut/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.magentys/donut)
-[![Javadoc](https://javadoc-emblem.rhcloud.com/doc/io.magentys/donut/badge.svg)](http://www.javadoc.io/doc/io.magentys/donut)
+[![Build Status](https://travis-ci.org/DonutReport/donut.svg?branch=master)](https://travis-ci.org/DonutReport/donut)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/report.donut/donut/badge.svg)](https://maven-badges.herokuapp.com/maven-central/report.donut/donut)
+[![Javadoc](https://javadoc-emblem.rhcloud.com/doc/report.donut/donut/badge.svg)](http://www.javadoc.io/doc/report.donut/donut)
 
-Donut is an open-source framework by the [MagenTys](https://magentys.io) team which is designed to produce clear and concise test execution reports over your unit, integration and acceptance tests.
-Donut currently supports any tool that produces gherkin json (ie. cucumber-jvm etc). For other frameworks like SpecFlow, JUnit, NUnit, you can use the adapters listed below to generate the gherkin jsons, which can then be processed by **donut**.
+Donut is an open-source framework by [MagenTys](https://magentys.io) & [MechanicalRock](https://mechanicalrock.io) and is designed to produce clear and concise test execution reports of your unit, integration and acceptance tests.

```

Comment:
Donut is an open-source framework by the teams at [MagenTys](https://magentys.io) & [MechanicalRock](https://mechanicalrock.io)

Reply:
[Mechanical Rock](https://mechanicalrock.io)

Reply:
Done